OSSM-8862 Migration Hub page content (#88651)
openshift/openshift-docs: migrating/ossm-migrating-from-service-mesh-2-to-3-assembly.adoc

:_mod-docs-content-type: ASSEMBLY
[id="ossm-migrating-from-service-mesh-2-to-3-assembly"]
= Migrating from Service Mesh 2 to Service Mesh 3
include::_attributes/common-attributes.adoc[]
:context: ossm-migrating-from-2-to-3-assembly

toc::[]

> The content contained within this migration section applies only if:

Reviewer: ```suggestion
The content in this migration section applies only in the following cases:
```

* You are an existing {SMProductName} user running {SMProduct} {SMv2Version}.
* You want to move to {SMProduct} 3.0.

[IMPORTANT]
====

> If you are not running {SMProduct} {smv2version}, you must update before you can continue. link:https://docs.redhat.com/en/documentation/openshift_container_platform/4.17/html/service_mesh/service-mesh-2-x#ossm-versioning_ossm-upgrade[Upgrading Service Mesh].

Reviewer: ```suggestion
If you are not running {SMProduct} {smv2version}, you must update before you can continue. For more information, see link:https://docs.redhat.com/en/documentation/openshift_container_platform/4.17/html/service_mesh/service-mesh-2-x#ossm-versioning_ossm-upgrade[Upgrading Service Mesh].
```
FYI, this link takes me to the versioning subsection within the "Upgrading Service Mesh" section.

Author: Ah, good catch. Various iterations specifically called out versioning and why it was important to understand how versioning works in 2.x (updating operator doesn't mean control plane updates, and a few caveats and things particular to 2.x) so it linked to the versioning subsection.

====

> From time to time during the migration process, you may need to reference, or will be directed to {SMProduct} 2.x content. It may be beneficial to open link:https://docs.redhat.com/en/documentation/openshift_container_platform/4.17/html/service_mesh/service-mesh-2-x[OpenShift Service Mesh 2.x] in a new tab or window for easier reference. This may be especially helpful when moving between {SMProduct} 2 `ServiceMeshControlPlane` resource content and {SMProduct} 3 `Istio` resource content.

Reviewer: ```suggestion
During the migration process, you might need to reference, or you might be directed to {SMProduct} 2.x content. It can be beneficial to open link:https://docs.redhat.com/en/documentation/openshift_container_platform/4.17/html/service_mesh/service-mesh-2-x[OpenShift Service Mesh 2.x] in a new tab or window for easier reference. This can be especially helpful when you move between {SMProduct} 2 `ServiceMeshControlPlane` resource content and {SMProduct} 3 `Istio` resource content.
```
- Avoid "may" in technical content: https://www.ibm.com/docs/en/ibm-style?topic=word-usage#may
